index.js

```diff
@@ -1,18 +1,47 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector("h1#header")
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "skyblue";
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+const travName = document.querySelector("#profile h2")
+travName.textContent = traveler.name
+
+const travNickName = document.querySelector("#profile em") 
+travNickName.textContent = traveler.nickname
 
+const travPhoto = document.querySelector("#profile img")
+travPhoto.src = traveler.photo
+travPhoto.alt = traveler.name 
 
 /***** Deliverable 4 *****/
+list = document.querySelector("#animals")
+
+traveler.animalSightings.forEach(function(obj){
+
+    const li = document.createElement("li")
+    li.setAttribute("data-id", obj.id)
```

Comment:
For dataset attributes, you could also use this syntax:

```js
li.dataset.id = animalObj.id
```

This will add a `data-id` attribute to the element like this:

```html
<li data-id="1">...</li>
```